### PR TITLE
fix java string format

### PR DESCRIPTION
**Pull request by acxz**
Just a small format fix:

Previous:
```
java,17.0.3"
```
After:
```
java,17.0.3
```

### CODE REVIEW

File: bin/versions.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ fi
 
 if [[ $LANGUAGES == *":java:"* ]]; then
     echo -n java,
-    java -version 2>&1 |grep "version" | cut -f3 -d " " | cut -c 2-9
+    java -version 2>&1 |grep "version" | cut -f2 -d "\""
```
Comment (waldyrious) [inline bin/versions.sh:27]: While you're at it, would you mind also adding a space between the pipe and `grep`? The inconsistency bothers me a bit :sweat_smile:

Comment (acxz) [reply bin/versions.sh:27]: haha sure!

Comment (acxz) [reply bin/versions.sh:27]: For some reason the github PR is not updating from the merging branch. Maybe it just needs some time to update

Comment (acxz) [reply bin/versions.sh]: ended up having to fp

**Pull request merged**